ibm/mas_devops/roles/suite_app_config/README.md

```diff
@@ -75,9 +75,44 @@ Optional. Name of the schema where Manage database lives in.
 ### mas_app_settings_demodata
 Optional. Flag indicating if manage demodata should be loaded or not.
 
-- Environment Variable: MAS_APP_SETTINGS_DEMODATA
+- Environment Variable: `MAS_APP_SETTINGS_DEMODATA`
 - Default: `false` (do not load demodata)
 
+### mas_app_base_language
```

Comment:
Just to keep consistent with other variables also used in application settings, can we use prefix `mas_app_settings_` instead of just `mas_app_` ? Same for the next two variables:

- `mas_app_settings_base_language`
- `mas_app_settings_secondary_languages`
- `mas_app_settings_server_bundles_size`

Reply:
fixed

Reply:
fixed!